packages/docs/docs/self-hosting/project-settings.md

```diff
@@ -50,6 +50,7 @@ The supported options that can be specified by a Super Admin in `Project.systemS
 | `graphqlMaxSearches`           | integer | The maximum number of searches allowed in a GraphQL query                                                                                                                 | none    |
 | `legacyFhirJsonResponseFormat` | boolean | If true, plain JSON formatting is incorrectly used instead of [FHIR JSON](https://hl7.org/fhir/R4/json.html) for some responses with content type `application/fhir+json` | false   |
 | `rateLimit`                    | integer | The maximum requests per minute allowed per IP address                                                                                                                    | 60000   |
-| `totalFhirQuota`               | integer | The maximum number of FHIR interactions that can be performed in a minute for any user in the project. See [Fhir Interaction Quota](/docs/rate-limits#fhir-interaction-quota). To enforce, set `enableFhirQuota` to true. | 50000   |
-| `enableFhirQuota`              | boolean | If true, the totalFhirQuota limit will be enforced, returning ```429 Too Many Requests``` errors when the limit is exceeded over a minute.                                                                                                 | false   |
+| `userFhirQuota`                | integer | The maximum number of FHIR interactions that can be performed in a minute by any single User in the project. See [Fhir Interaction Quota](/docs/rate-limits#fhir-interaction-quota). To enforce, set `enableFhirQuota` to true. | 50000   |
```

Comment:
The part about "maximum number of FHIR interactions" isn't totally correct: these limits are expressed in abstract "quota units", which are essentially a weighted sum of the different interaction types